src/devices/tuya.ts

```diff
@@ -4229,22 +4229,41 @@ const definitions: Definition[] = [
         description: 'Wall-mount thermostat',
         fromZigbee: [tuya.fz.datapoints],
         toZigbee: [tuya.tz.datapoints],
-        onEvent: tuya.onEvent({timeStart: '1970'}),
+        onEvent: tuya.onEventSetLocalTime,
```

Comment:
Why was this change needed?

Reply:
Without this change, ZWT07 and zigbee2mqtt enter infinite loop with :
<code>No converter available for 'ZWT07' with cluster 'manuSpecificTuya' and type 'commandMcuVersionResponse' and data '{"seq":28928,"version":65}'</code>
See example output : https://gist.github.com/tvoyat/c70369ae3093f217ff9bf2691bedee0f
I don't know why it doesn't do it anymore with `tuya.onEventSetLocalTime`...